src/utils/buildSearchQuery.ts

```diff
@@ -37,7 +37,7 @@ export default function buildSearchQuery(
     query.sort_field = routerQuery[QUERY_PARAMS.sort_field]?.toString();
   }
 
-  // If no sort order is provided, and we are in "browse" mode, we want to set the sort order to latest date "date:desc"
+  // If no sort order is provided, and we are in "browse" mode (i.e. no search term), we want to set the sort order to latest date "date:desc"
```

Comment:
I know I asked previously, I do think "browse" adds more complexity to understanding simply not having a `q` set.

Reply:
That's fair, with this comment I am trying to give some context around the _why_ because it is clearer from the code of the what